.github/workflows/ci-only.yaml

```diff
@@ -0,0 +1,67 @@
+name: ci-only
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "master"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        uses: actions/checkout@master
+        with:
+            fetch-depth: 1
+      -  
+        name: Get tags
+        id: get_tag
+        run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+      - 
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - 
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - 
+        name: Local docker build (non-root image)
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          push: false
+          platforms: linux/amd64
+          target: release
+          build-args: |
+            VERSION=${{ steps.get_tag.outputs.TAG }}
+            GIT_COMMIT=${{ github.sha }}
+          load: true
+          tags: openfaas/faas-cli:${{ github.sha }}
+      - 
+        name: Local docker build (root image)
+        uses: docker/build-push-action@v2
+        with:
+          context: .
```

Comment:
Context + path can be dropped.

Reply:
Isn't there a new Dockerfile for this like Dockerfile.nonroot?

OK you are controlling it via `--target release` here -> https://github.com/openfaas/faas-cli/blob/master/build.sh

Reply:
@alexellis I'm controlling it with
```yaml
with:
   context: .
   file: ./Dockerfile
   target: release
```
and 
```yaml
with:
   context: .
   file: ./Dockerfile
   target: root
```